ggp-engine/PhysicsManager.cpp

```diff
@@ -0,0 +1,137 @@
+#include "PhysicsManager.h"
+#include "RigidBody.h"
+#include "Spatial.h"
+
+using namespace DirectX;
+using namespace Physics;
+
+PhysicsManager* PhysicsManager::Instance = nullptr;
+
+const UINT PhysicsManager::GetRigidBodyCount() const
+{
+    return rBodyCount;
+}
+
+const bool Physics::PhysicsManager::Raycast( const DirectX::XMFLOAT3 & aOrigin, const DirectX::XMFLOAT3 & aDirectoin, const float distance )
+{
+    printf( "\tStart raycast!\n" );
```

Comment:
assuming this is going to change into real raycast code

Reply:
Correct